### DISCUSSION TITLE
Rack import missing rack type option

### DISCUSSION THREAD

**Discussion by chirpp2** [netbox-community/netbox#18148] [Q&A]
We are adding rack types to use for when we manually are adding racks to the system. We also want to have the ability to import racks and the rack type field is not allowed. Am i missing something on this? 

When i choose the import option that field does not show as an option, and if i try to add a field rack_type to my csv it errors.

version 4.1.6

**Comment by atownson** [comment]
The `rack_type` field is missing from the bulk import form. I would suggest creating a feature request (or perhaps bug) issue to have the field added.

**Reply by chirpp2** [reply]
Sounds good. Thanks!

**Reply by pl0xym0r** [reply]
#17465
